.github/workflows/ci.yaml

```diff
@@ -61,23 +61,22 @@ jobs:
           echo "::set-output name=tag::$imageTag"
           echo "::set-output name=repo::ghcr.io/${{ github.repository }}"
       - name: Push Dev Tag
-        if: startsWith(github.ref, 'refs/heads/dev')
```

Comment:
Does this mean we'll end up tagging twice if github.ref is 'refs/heads/main'?

Reply:
> the image job runs anyhow just for main and dev

ah, nevermind

Reply:
the job runs for both main and dev and each will create its own docker tag. For dev, it is `:dev` for main it is `:latest` and `:v${currentVersion}`